docs/files.md

```diff
@@ -0,0 +1,97 @@
+# Input and Output Files
+
+## constraints.txt
+
+The `constraints.txt` is the input file used to specify the exact version constraints of python packages that are needed to build a wheel. This file is similar to `requirements.txt` but instead of listing packages that must be installed, it sets constraints on versions of packages, ensuring that certain versions are either used or avoided during the wheel building process.
```

Comment:
```suggestion
The `constraints.txt` is the input file used to control the resolution of python packages that are needed to build a wheel. This file is similar to `requirements.txt` but instead of listing packages that must be installed, it sets constraints on versions of packages, ensuring that certain versions are either used or avoided during the wheel building process.
```

We do tend to use it to set specific versions, but it's also possible to just use `!=` rules to avoid packages.

Reply:
Okay, changed it!